lib/migrator/methods/_getMigrationTypeParams.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports =
+	function _getMigrationTypeParams(migrationFileType = 'executable') {
```

Comment:
Maybe it should be called `_getMigrationFileTypeParams`? since it accepts `migrationFileType`